src/AddOn.Episerver.Settings/Core/ISettingsService.cs

```diff
@@ -58,20 +58,48 @@ public interface ISettingsService
     IEnumerable<ContentReference> SettingsRoots { get; }
 
     /// <summary>
-    ///     Gets the settings.
+    ///     Gets a setting, starting the search for it from the current content context.
     /// </summary>
     /// <typeparam name="T">The settings type</typeparam>
     /// <returns>An instance of <typeparamref name="T" /></returns>
-        T GetSettings<T>() where T : SettingsBase;
+    T GetSetting<T>() where T : SettingsBase;
 
     /// <summary>
-    ///     Gets the settings.
+    ///     Gets a setting, starting the search from the provided content reference.
```

Comment:
Perhaps clarify that this uses recursion behind the scenes:

"Gets the first matching setting by traversing the content tree, starting..."

Reply:
fixed